app/models/customer.rb

```diff
@@ -0,0 +1,6 @@
+class Customer < ApplicationRecord
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+
+  has_many :invoices, dependent: :destroy
```

Comment:
will have to double check on this but rubocop wanted a dependent clause at the end.
in theory, it makes sense why i would need it